features/evaluation_spec.rb

```diff
@@ -9,6 +9,14 @@
   end
   include_context 'simple git repository'
 
+  let(:map_generator_config) do
```

Comment:
```
map_generator_config do
  "STRING"
end
```
will be a bit more declarative

Reply:
I don't see much value in it, especially with the added complexity of handling overriding in nested contexts.

Reply:
there is no complexity. its literally more explicit alias.
```
def map_generator_config(&block)
  let(:map_generator_config, &block)
end
```